driver.md

```diff
@@ -86,14 +126,14 @@ To do so, we'll extend sort `JSON` with some EVM specific syntax, and provide a
            <sigR>       TR   </sigR>
            <sigS>       TS   </sigS>
            <data>       DATA </data>
-           <txType>     TYPE </txType>
            <txChainID>  CID  </txChainID>
            <txAccess>   TA   </txAccess>
+           <txType> AccessList </txType>
            ...
          </message>
 
-    syntax EthereumCommand ::= loadTx ( Account )
- // ---------------------------------------------
+    syntax EthereumCommand ::= loadTx ( AcctExp ) [strict]
+ // ------------------------------------------------------
```

Comment:
With `#TxSender` as a contextual function for sort `Account`, then we don't need this to be `strict`.